python/ray/dashboard/subprocesses/tests/test_e2e.py

```diff
@@ -206,13 +195,17 @@ async def test_logging_in_module(aiohttp_client, default_module_config):
     app = await start_http_server_app(default_module_config, [TestModule])
     client = await aiohttp_client(app)
 
-    response = await client.post("/logging_in_module", data=b"")
+    response = await client.post(
+        "/logging_in_module", data=b"Not all those who wander are lost"
+    )
     assert response.status == 200
     assert await response.text() == "done!"
 
     # Assert the log file name and read the log file
-    log_file_path = default_module_config.log_dir / "dashboard-TestModule.log"
-    with open(log_file_path, "r") as f:
+    log_file_path = (
+        pathlib.Path(default_module_config.log_dir) / "dashboard-TestModule.log"
```

Comment:
can we change log file name to `dashboard_xxx.log`, just to be consistent with the other log files.